scripts/ci/run_perftest.sh

```diff
@@ -94,20 +105,29 @@ datestring=$(date +%s)
 seed=$datestring
 db_name="benchmark_db"
 
+# Controls how many data points will be generated. The total number of points
```

Comment:
The comment about total number of points being `scale_var * scale_var` is only applicable to the metaquery data generator use case - for other use cases it's different. Could probably reduce this comment to `Controls how many data points will be generated.` and leave it at that.